src/DssProxyActions.sol

```diff
@@ -427,38 +572,49 @@ contract DssProxyActions is Common {
 
     // Deprecated
     function safeLockGem(
-        address manager,
+        address manager_,
         address gemJoin,
         uint256 cdp,
         uint256 amt,
         bool,
         address owner
     ) public {
-        safeLockGem(manager, gemJoin, cdp, amt, owner);
+        require(manager_ == address(manager), "DssProxyActions/Non-matching-manager");
+        safeLockGem(gemJoin, cdp, amt, owner);
     }
     //
 
     function safeLockGem(
-        address manager,
         address gemJoin,
         uint256 cdp,
         uint256 amt,
         address owner
     ) public {
         require(ManagerLike(manager).owns(cdp) == owner, "owner-missmatch");
-        lockGem(manager, gemJoin, cdp, amt);
+        lockGem(gemJoin, cdp, amt);
```

Comment:
No need for ManagerLike casting on line 593

Reply:
Done.